crates/sui-benchmark/tests/simtest.rs

```diff
@@ -79,6 +79,18 @@ mod test {
         test_simulated_load(TestInitData::new(&test_cluster).await, 60).await;
     }
 
+    #[sim_test(config = "test_config()")]
+    async fn test_simulated_load_with_reconfig_and_correlated_crashes() {
+        sui_protocol_config::ProtocolConfig::poison_get_for_min_version();
+
+        register_fail_point_if("correlated-crash-after-consensus-commit-boundary", || true);
+        // TODO: enable this - right now it causes rocksdb errors when re-opening DBs
```

Comment:
The error is that we are failing to acquire the lock when re-opening the DB, so it may be that we leak a rocksdb handle when crashing here. It will take some time to investigate this

Reply:
So there could be leaked sui-node tasks that do not shutdown on panic and they hold rocksdb handles?

Reply:
actually are we talking about the sui-node rocksdb or the consensus one @mystenmark  (or both?)?

Reply:
I think sui-node. If you'd like to investigate just uncomment this and see what happens 🗡️